packages/admin-panel/src/authentication/LoginForm.js

```diff
@@ -108,11 +100,10 @@ const mergeProps = (stateProps, { dispatch }, ownProps) => ({
   ...ownProps,
   ...stateProps,
   onChangeEmailAddress: event => dispatch(changeEmailAddress(event.target.value)),
-  onChangePassword: event => dispatch(changePassword(event.target.value)),
   onChangeRememberMe: event => dispatch(changeRememberMe(event.target.checked)),
-  onLogin: event => {
+  onLogin: (event, password) => {
     event.preventDefault();
-    dispatch(login(stateProps.emailAddress, stateProps.password));
+    dispatch(login(stateProps.emailAddress, password));
```

Comment:
Thanks for making this change. Hopefully we can steadily transition most of the app state out of redux and into it's relevant components 👍